benchmarks/maxtext_v5p_model_configs.py

```diff
@@ -104,4 +104,78 @@
             + xla_flags_library.DATA_PARALLEL_OVERLAP
         ),
     ),
-)
\ No newline at end of file
+)
+
+llama2_70b_v5p_128 = _add_to_model_dictionary(
+  v5p_model_dict,
+  MaxTextModel(
+    model_name="llama2_70b_v5p_128",
+    model_type="llama2-70b",
+    tuning_params={
+        "ici_fsdp_parallelism": -1,
+        "per_device_batch_size": 4,
+        "remat_policy": "save_dot_except_mlpwi",
+        "max_target_length": 4096,
+        "use_iota_embed": True,
+        "dataset_path": "gs://max-datasets-rogue",
+        "dataset_type": "synthetic",
+        "reuse_example_batch": 1,
+        "enable_checkpointing": False,
+        "profiler": "xplane",
+    },
+    xla_flags=(
+        xla_flags_library.DATA_PARALLEL_OVERLAP
+        + xla_flags_library.CF_FOR_ALL_GATHER
+    ),
+	)
+)
+
+llama2_7b_v5p_128 = _add_to_model_dictionary(
+  v5p_model_dict,
+  MaxTextModel(
+    model_name="llama2_7b_v5p_128",
+    model_type="llama2-7b",
+    tuning_params={
+        "ici_fsdp_parallelism": -1,
+        "per_device_batch_size": 4,
+        "remat_policy": "minimal",
+        "max_target_length": 4096,
+        "use_iota_embed": True,
+        "tokenizer_path": os.path.join("assets", "tokenizer.llama2"),
+        "dataset_path": "gs://max-datasets-rogue",
+        "dataset_type": "synthetic",
+        "reuse_example_batch": 1,
+        "enable_checkpointing": False,
+        "profiler": "xplane",
+    },
+    xla_flags=(
+        xla_flags_library.DATA_PARALLEL_OVERLAP
+        + xla_flags_library.CF_FOR_ALL_GATHER
+    ),
+	)
+)
+
+gpt_3_175b_v5p_128 = _add_to_model_dictionary(
+  v5p_model_dict,
+  MaxTextModel(
+    model_name="gpt_3_175b_v5p_128",
+    model_type="gpt3-175b",
+    tuning_params={
+        "ici_fsdp_parallelism": -1,
+        "ici_tensor_parallelism": 2,
+        "per_device_batch_size": 0.5,
+        "allow_split_physical_axes":True,
+        "remat_policy": "save_dot_with_context_except_mlp",
+        "max_target_length": 4096,
+        "attention": "flash",
+        "dataset_type": "synthetic",
+        "reuse_example_batch": 1,
+        "enable_checkpointing": False,
+        "profiler": "xplane",
+    },
+    xla_flags=(
+        xla_flags_library.DATA_PARALLEL_OVERLAP
+        + xla_flags_library.CF_FOR_ALL_GATHER
+    ),
+	)
+)
```

Comment:
can you also add gpt_3_175b_v5p_128_sc config which runs with SC offload? you can use `ENABLE_SPARSECORE_OFFLOADING_FOR_ALL_GATHER` to replace `CF_FOR_ALL_GATHER `